public/js/components/SplitBox.css

```diff
@@ -14,7 +14,7 @@
 .split-box .controlled {
   display: flex;
   overflow: auto;
-  transition: width 0.5s ease;
+  /*transition: width 0.5s ease;*/
```

Comment:
oh, this is probably for the collapse button. But it is firing for all of the drag handler